tools/integration_tests/util/operations/file_operations.go

```diff
@@ -295,6 +295,31 @@ func ReadChunkFromFile(filePath string, chunkSize int64, offset int64) (chunk []
 	return
 }
 
+// Write data of chunkSize in file at given offset.
+func WriteChunkSizeInFile(file *os.File, chunkSize int, offset int64) error {
+	chunk := make([]byte, chunkSize)
+	_, err := rand.Read(chunk)
+	if err != nil {
+		return fmt.Errorf("error while generating random string: %s", err)
+	}
+
+	// Write data in the file.
+	n, err := file.WriteAt(chunk, offset)
+	if err != nil {
+		return fmt.Errorf("Error in writing randomly in file:%v", err)
+	}
+	if n != chunkSize {
+		return fmt.Errorf("Incorrect number of bytes written in the file actual %d, expected %d", n, chunkSize)
+	}
+
+	err = file.Sync()
```

Comment:
Why do we need this sync here after each chunk write ?

Reply:
We want to sync each chunk to gcs in case of [random write](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/write_large_files/random_write_large_file_test.go#L77)